packages/bodiless-organisms/src/components/Menu/MegaMenu.tsx

```diff
@@ -97,30 +94,29 @@ const asMenuBase = (nodeKeys?: WithNodeKeyProps) => flow(
     Item: asChameleonSubList,
   }),
   withSubMenuDesign(asMenuSubList),
+  withMenuDesign({
+    Item: withoutProps(['addItem', 'deleteItem', 'canDelete']),
+  }),
 );
 
 // Next we replace basic list elements with rc-menu elements to create a menu.
 
 // Applies above designs to the chameilion sublist
 const asChameleonSubMenu = withDesign({
-  Basic: flowRight(
```

Comment:
As for simple menu, it would be great to get rid of all of this.  Assuming we can kill 'asSubMenu` (by making the sublist wrapper design elements stylable in the base menu list), the main thing left here is the tout title replacement, and i think we can just remove this since the tout tile is always specified on the site level anyway.

Reply:
Done